fastlane_bot/tools/cpc.py

```diff
@@ -1555,6 +1700,46 @@ def isbigger(self, small, big):
             return big > self.EPS
         return big / small > 1 + self.EPS
```

Comment:
**Not directly related to this PR**.

The description below assumes `big, small >= 0` (otherwise this function might produce undesired output).

Assuming that the desired result in the general case here is to return check if the ratio between `big` and `small` is larger than one plus epsilon, the special case of `small == 0` should just return `big > 0`.

Note that it also works out mathematically:
- `big > 0 and small == 0` -->  `big / small  == inf > 1 + self.EPS`
- `big == 0 and small == 0` -->  `big / small tends to 1 < 1 + self.EPS`

That being said, I suspect the correct behavior to desire is slightly different, and would be achieved via:
```
return big > self.EPS + small
```
Again, assuming that both of these values are non-negative.

Reply:
> the special case of small == 0 should just return big > 0.

not quite sure I am using it at all, but the idea here is that for small == big you are guaranteed False

Reply:
not sure the case ``small == 0`` is particularly relevant, but I did want to handle it
thing is -- normally I want EPS to be a dimensionless constant because ``big > small + self.EPS`` means different things then ``small = 1e-20`` vs ``small = 1e20``. That is the case that I want to handle well -- especially in crypto where we can have prices in both orders of magnitude.

The case `small=0` is an afterthought and given that EPS will the 1e-3 ... 1e-9 it seems good enough